src/urllib3/util/ssl_.py

```diff
@@ -116,6 +118,7 @@ def _const_compare_digest_backport(a, b):
 try:
     from ssl import SSLContext  # Modern SSL?
 except ImportError:
+    IS_SSL_SESSION_SUPPORTED = False
```

Comment:
Probably should delete the other commit which disables session tickets.

The two don't conflict, this commit overwrites it in practice but it's not elegant.